### PR TITLE
Spelling correction

### PR DESCRIPTION
**Pull request by Ryan-Irey**
Updated "interrests" to "interests" on Line 291 of organization.rst

### CODE REVIEW

File: docs/source/i523/2017/organization.rst [hunk 1]
```diff
@@ -75,6 +81,8 @@ EOrganization.5:
              the end of Tuesday, meaning you get feedback back on
              Wednesday. Thus dependent on when you ask it may take 4
              days for an answer.
+=======
+>>>>>>> Spelling correction
```
Comment (ssheybani) [inline docs/source/i523/2017/organization.rst]: This >>>>> is an error and needs to be removed. I guess you have copied it from the command line terminal by mistake.

File: organization.rst [hunk 2]
```diff
@@ -0,0 +1,278 @@
+Organization
```
Comment (ssheybani) [inline organization.rst]: This whole file looks redundant and also cannot be in the main directory anyway. Please remove it.

**Comment by ssheybani** [comment]
@Ryan-Irey Thank you for your contribution. 
However, there are a few small errors that need to be fixed before it can be merged. I commented them.

Make sure that your changes show correctly by running make and make view before creating a pull request.

**Comment by laszewsk** [comment]
this is a conflict

**Pull request closed without merging**